apps/judicial-system/backend/infra/judicial-system-backend.ts

```diff
@@ -40,8 +40,8 @@ export const serviceSetup = (): ServiceBuilder<'judicial-system-backend'> =>
       SQS_REGION: 'eu-west-1',
       BLOCKED_API_INTEGRATION: {
         dev: '',
-        staging: 'COURT,POLICE_DOCUMENT,POLICE_CASE',
-        prod: 'POLICE_CASE',
+        staging: 'COURT,COURT_LITIGANTS,POLICE_DOCUMENT,POLICE_CASE',
+        prod: 'COURT_LITIGANT,POLICE_CASE',
```

Comment:
I see there is a difference in the names of the integrations, S missing in the prod statement compared to staging. Is this intentional?

Reply:
Not intentional, will fix.

Reply:
Well spotted